numba/pycc/modulemixin.c

```diff
@@ -112,6 +116,14 @@ PYCC(pycc_init_) (PyObject *module, PyMethodDef *defs,
                                     env_def_t *envs,
                                     env_gv_t *envgvs)
 {
+    /* Aligns hashsecret with values in current python process so that
+     * hashes computed inside pycc module are the same as in the importing
+     * python process.
+     */
```

Comment:
```suggestion
    /* Aligns hashsecret with values in current python process so that
     * hashes computed inside the pycc module are correct if imported
     * by the current process. Subsequent imports get the right hash
     * secret through `numba.cpython.hashing._load_hashsecret`.
     */
```